File: C/Iniciante/1154 - Idades..c
/*
* Autor: Pierre Vieira
* Data de submissão: 16/02/2019 21:18:24
*/
#include<stdio.h>
int main(){
    int soma = 0, n = 0;
    float cont = 0;
    while (n>=0){
        scanf("%d", &n);
        if (n >= 0){
            soma += n;
            cont ++;
        }
    }
    printf("%.2f\n", soma/(cont));
}

File: C#/Iniciante/1154 - Idades.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 16/05/2020 21:49:40
*/
using System;
using System.Globalization;

namespace URI {
    class Program {
        static void Main(string[] args) {
            int idade, soma_idades, cont;
            double media;
            cont = soma_idades = 0;
            while (true) {
                idade = int.Parse(Console.ReadLine());
                if (idade < 0) {
                    break;
                }
                soma_idades += idade;
                cont += 1;
            }
            media = soma_idades / (double) cont;
            Console.WriteLine(media.ToString("F2", CultureInfo.InvariantCulture));
        }
    }
}
